packages/dashboard/src/store/index.ts

```diff
@@ -1,25 +1,16 @@
 import { Store } from 'redux';
 import { configureStore } from '@reduxjs/toolkit';
-import createSagaMiddleware from 'redux-saga';
```

Comment:
remove redux-saga. Orignial functionality will be added back in another way after ongoing refactoring.